core/src/main/resources/org/apache/spark/ui/static/stagepage.js

```diff
@@ -0,0 +1,926 @@
+    $('#shuffle_read_blocked_time').attr("data-toggle", "tooltip")
+        .attr("data-placement", "bottom")
+        .attr("title", "Time that the task spent blocked waiting for shuffle data to be read from remote machines.");
+    $('#shuffle_remote_reads').attr("data-toggle", "tooltip")
+        .attr("data-placement", "bottom")
+        .attr("title", "Total shuffle bytes read from remote executors. This is a subset of the shuffle read bytes; the remaining shuffle data is read locally. ");
+    $('#result_serialization_time').attr("data-toggle", "tooltip")
+            .attr("data-placement", "bottom")
+            .attr("title", "Time spent serializing the task result on the executor before sending it back to the driver.");
+    $('#getting_result_time').attr("data-toggle", "tooltip")
+            .attr("data-placement", "bottom")
+            .attr("title", "Time that the driver spends fetching task results from workers. If this is large, consider decreasing the amount of data returned from each task.");
+    $('#peak_execution_memory').attr("data-toggle", "tooltip")
+            .attr("data-placement", "bottom")
+            .attr("title", "Execution memory refers to the memory used by internal data structures created during " +
+                "shuffles, aggregations and joins when Tungsten is enabled. The value of this accumulator " +
+                "should be approximately the sum of the peak sizes across all such data structures created " +
+                "in this task. For SQL jobs, this only tracks all unsafe operators, broadcast joins, and " +
+                "external sort.");
+    $('#scheduler_delay').tooltip(true);
+    $('#task_deserialization_time').tooltip(true);
+    $('#shuffle_read_blocked_time').tooltip(true);
+    $('#shuffle_remote_reads').tooltip(true);
+    $('#result_serialization_time').tooltip(true);
+    $('#getting_result_time').tooltip(true);
+    $('#peak_execution_memory').tooltip(true);
+    tasksSummary = $("#active-tasks");
+    getStandAloneAppId(function (appId) {
+
+        var endPoint = stageEndPoint(appId);
+        $.getJSON(endPoint, function(response, status, jqXHR) {
+
+            var responseBody = response[0];
+            // prepare data for task aggregated metrics table
+            indices = Object.keys(responseBody.executorSummary);
+            var task_summary_table = [];
+            indices.forEach(function (ix) {
+               responseBody.executorSummary[ix].id = ix;
+               task_summary_table.push(responseBody.executorSummary[ix]);
+            });
+
+            // prepare data for accumulatorUpdates
+            var indices = Object.keys(responseBody.accumulatorUpdates);
+            var accumulator_table_all = [];
+            var accumulator_table = [];
+            indices.forEach(function (ix) {
+               accumulator_table_all.push(responseBody.accumulatorUpdates[ix]);
+            });
+
+            accumulator_table_all.forEach(function (x){
```

Comment:
get rid of accumulator_table_all by combining loop.  array filter method may work here

Reply:
Done